pkg/plugins/policies/core/rules/testdata/sort/by-origin.input.yaml

```diff
@@ -0,0 +1,45 @@
+# MeshTimeout with targetRef to the real MeshService resource
+type: MeshTimeout
+name: matched-for-rules-ccc
+mesh: mesh-1
+labels:
+  kuma.io/origin: zone
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        connectionTimeout: 111s
+        idleTimeout: 109s
+---
+type: MeshTimeout
+name: matched-for-rules-bbb
+mesh: mesh-1
```

Comment:
can we have a policy without `kuma.io/origin` label?

Reply:
I don't remember how exactly you can have it, but I'm pretty sure it's possible. Maybe if you upgrade from the version that doesn't add `kuma.io/origin` automatically and you had the origin validation disabled then I guess after the Kuma CP upgrade you'll have resources without origin

Reply:
this sounds like a possible use case